ui.pyw

```diff
@@ -543,6 +550,7 @@ class EditRestaurantDialogue(RestaurantDialogue, object):
 
     def fill_initial_values(self):
         self.restaurant_text.insert(END, self.restaurant_name)
+        self.comment_text.insert(END, self.restaurant_comment)
```

Comment:
The idea was to display it on double click in the restaurant tab, but since that is not (yet) a list I left it for a future commit.